### ISSUE TITLE
‘Shopify’ tool crashed with sanity 2.0

### ISSUE THREAD

**Issue by sndrgb** [good-idea/sane-shopify#123]
Hi @good-idea,

I update sanity packages from 1.150.5 to 2.0.4 and now the shopify tab gave me this:

![Screenshot 2020-10-15 at 18 37 31](https://user-images.githubusercontent.com/1871529/96159982-a3193480-0f15-11eb-96a1-ed923318eddf.png)

```
Error: No context provided for reporter. Make sure that the component calling "useReporter(field-, ...)", is wrapped in a <Tracker> element
    at useReporterGuard (/static/js/app.bundle.js:231350:9)
    at http://localhost:3333/static/js/app.bundle.js:231448:7
    at commitHookEffectListMount (/static/js/vendor.bundle.js:23231:26)
    at commitLifeCycles (/static/js/vendor.bundle.js:23287:9)
    at commitLayoutEffects (/static/js/vendor.bundle.js:26301:7)
    at HTMLUnknownElement.callCallback (/static/js/vendor.bundle.js:3655:14)
    at Object.invokeGuardedCallbackDev (/static/js/vendor.bundle.js:3704:16)
    at invokeGuardedCallback (/static/js/vendor.bundle.js:3759:31)
    at commitRootImpl (/static/js/vendor.bundle.js:26039:9)
    at unstable_runWithPriority (/static/js/vendor.bundle.js:29259:12)
```

Any ideas?
Can I help?

Thanks in advance

**Comment by good-idea** [comment]
I'll have a fix coming shortly! in the meantime, the only option is to downgrade your sanity installation to 1.x

**Comment by liqueflies** [comment]
Any updates on this?

**Comment by good-idea** [comment]
@liqueflies apologies, I'm a little behind! I was going to include it in a larger refactor, but that's taking more time than expected so i'll make a standalone patch in the meantime. I'll ping you here when it's up.

**Comment by good-idea** [comment]
Fixed in 0.18.0!
